lambdas/checklist/serverless.yml

```diff
@@ -71,5 +70,16 @@ functions:
               paths:
                 id: true
 
+  publish-evaluation:
```

Comment:
Maybe evaluation-publisher might be better. The other lambda that does processing can be named evaluation-processor.